bpf-gpl/conntrack.h

```diff
@@ -22,15 +22,10 @@
 #include "nat.h"
 #include "bpf.h"
 #include "icmp.h"
+#include "types.h"
 
 // Connection tracking.
 
-struct calico_ct_key {
```

Comment:
Have moved types from `xxx.h` to `xxx_types.h` so that they can be imported and used in the context structs without circular deps.